src/HdInsightOnAks/HdInsightOnAks.Autorest/README.md

```diff
@@ -49,14 +46,15 @@ In this directory, run AutoRest:
 ### General settings
 > Values
 ``` yaml
-commit: f09aacf4c6b63be416212cb182f6b31e8bc6d545
+tag: package-preview-2023-11
+commit: 754d31188c080f20b8155acdf8b8fbd77dace467
 require:
   - $(this-folder)/../../readme.azure.noprofile.md
   - $(repo)/specification/hdinsight/resource-manager/Microsoft.HDInsight/HDInsightOnAks/readme.md
 
 # input-file:
-#  - D:\source\azure-rest-api-specs\specification\hdinsight\resource-manager\Microsoft.HDInsight\HDInsightOnAks\preview\2023-06-01-preview\hdinsight.json
-inlining-threshold: 100
+#  - D:\HDI-Repo\azure-rest-api-specs\specification\hdinsight\resource-manager\Microsoft.HDInsight\HDInsightOnAks\preview\2023-11-01-preview\hdinsight.json
```

Comment:
Local input file is not qualified. Please replace with a link from the swagger repo.

Reply:
It's just tested locally, and it's been commented out. There is commit id on it to get the file from swagger repo.
![image](https://github.com/Azure/azure-powershell/assets/58678302/926b3719-56a3-421b-904a-70f444a0c88d)

Reply:
Please remove the test location from the file. Thanks!

Reply:
Removed, thanks.